src/components/tasks/card/index.tsx

```diff
@@ -1,160 +1,209 @@
-        ${isTaskOverdue() && classNames.overdueTask}
-    `}
-    >
-      <div className={classNames.cardItems}>
-        <span
-          className={classNames.cardTitle}
-          contentEditable={shouldEdit}
-          onKeyPress={(e) => handleChange(e, 'title')}
-          role="button"
-          tabIndex={0}
-        >
-          {cardDetails.title}
-        </span>
-        <span>
-          <span className={classNames.cardSpecialFont}>Status:</span>
-          <span
-            className={classNames.cardStatusFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'status')}
-            style={{ color: statusFontColor }}
-            role="button"
-            tabIndex={0}
-          >
-            {cardDetails.status}
-          </span>
-        </span>
-      </div>
-      <div className={classNames.cardItems}>
-        <span>
-          <Image
-            src="/calendar-icon.png"
-            alt="calendar icon"
-            width={iconWidth}
-            height={iconHeight}
-          />
-          <span className={classNames.cardSpecialFont}>Due Date</span>
-          <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'endsOn')}
-            role="button"
-            tabIndex={0}
-          >
-            {fromNowEndsOn}
-          </span>
-        </span>
-      </div>
-      <div className={classNames.cardItems}>
-        <span
-          className={classNames.cardSpecialFont}
-          contentEditable={shouldEdit}
-          onKeyPress={(e) => handleChange(e, 'startedOn')}
-          role="button"
-          tabIndex={0}
-        >
-          Started
-          {' '}
-          {fromNowStartedOn}
-        </span>
-        <span>
-          <span className={classNames.cardSpecialFont}>Assignee:</span>
-          <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'assignee')}
-            role="button"
-            tabIndex={0}
-          >
-            {cardDetails.assignee}
-          </span>
-          <span
-            className={classNames.contributorImage}
-          >
-            <Image
-              src={assigneeProfilePic}
-              alt="Assignee profile picture"
-              onError={contributorImageOnError}
-              width={45}
-              height={45}
-            />
-          </span>
-        </span>
-      </div>
-    </div>
-  );
-};
-
-export default Card;
+import { FC, useState } from 'react';
+import Image from 'next/image';
+import classNames from '@/components/tasks/card/card.module.scss';
+import task from '@/interfaces/task.type';
+import { AVAILABLE, BLOCKED, COMPLETED, VERIFIED } from '@/components/constants/beautified-task-status';
+
+const moment = require('moment');
+
+type Props = {
+  content: task;
+  shouldEdit: boolean;
+  onContentChange: any;

```

Comment:
Instead of any Could we try creating type?
`type OnContentChangeFunction = (string,Object) => void;`
And then we could pass instead of any?